File: examples/src/Presentations/Properties/InsertSvgIntoPresentation.cs
using Aspose.slides.Export

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
when the project is build. Please check https:#docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http:#www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http:#www.aspose.com/community/forums/default.aspx
*/


namespace Aspose.slides.Examples.CSharp.Presentations
{
    public class InsertSvgIntoPresentation
    {
        public static void Run()
        {
            #ExStart:InsertSvgIntoPresentation
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_PresentationProperties()
            using (p = new Presentation())
         {
            svgContent = File.ReadAllText(svgPath)
            emfImage = p.Images.AddFromSvg(svgContent)
            p.slides[0].shapes.AddPictureFrame(ShapeType.Rectangle, 0, 0, emfImage.width, emfImage.height, emfImage)
            p.save(outPptxPath, slides.export.SaveFormat.PPTX)

           
            }
            #ExEnd:InsertSvgIntoPresentation
        }
    }
}

File: examples/src/Presentations/Properties/InsertSvgIntoPresentation.py
import aspose.slides as slides

def insert_svg():
    #ExStart:InsertSvgIntoPresentation
    # The path to the documents directory.

    dataDir = "./examples/data/"
    outDir = "./examples/out/"


    with slides.Presentation() as p:
        with open(dataDir + "image3.svg", "rb") as file:
            svgContent = file.read()

        svgImage = slides.SvgImage(svgContent)
        ppImage = p.images.add_image(svgImage)
        p.slides[0].shapes.add_picture_frame(slides.ShapeType.RECTANGLE, 0, 0, ppImage.width, ppImage.height, ppImage)
        p.save(outDir + "insert_svg_out.pptx", slides.export.SaveFormat.PPTX)

    
    #ExEnd:InsertSvgIntoPresentation
